graylog2-web-interface/src/views/components/aggregationwizard/sort/SortsConfiguration.tsx

```diff
@@ -57,13 +57,11 @@ const SortsConfiguration = () => {
         <SortableList
           items={sort}
           onMoveItem={(newSort) => setFieldValue('sort', newSort)}
-          customListItemRender={({ item, index, dragHandleProps, draggableProps, className, ref }) => (
+          customListItemRender={({ item, index, dragHandle, className, ref }) => (
```

Comment:
To fix this, we can create a component `CustomListItemRender` with props `{ item, index, dragHandle, className, ref } ` outside of the main and put inside `removeSort` callback and return whole this render part